src/bin/cql-stress-cassandra-stress/operation/row_generator.rs

```diff
@@ -0,0 +1,147 @@
+use scylla::_macro_internal::CqlValue;
+
+use crate::{
+    java_generate::{
+        distribution::{fixed::FixedDistribution, sequence::SeqDistribution, Distribution},
+        values::{Blob, Generator, GeneratorConfig, HexBlob},
+    },
+    settings::CassandraStressSettings,
+};
+use std::sync::Arc;
+
+use super::recompute_seed;
+
+/// A row generator structure.
+///
+/// Row-generation logic:
+/// - sample the `pk_seed` from `pk_seed_distribution`
+/// - seed the `pk_generator` with sampled `pk_seed`
+/// - generate the partition key with `pk_generator`
+/// - compute the seed for the `column_generators` based on generated pk
+/// - generate the rest of the row (seeding the `column_generators` with computed seed)
+///
+/// I think it's a great place to address how read and write workloads cooperate.
+/// For reference, see: https://github.com/scylladb/cql-stress/pull/43#discussion_r1304274035.
+///
+/// At first, we need to notice that as long as `pk_seed_distribution` is a DETERMINISTIC distribution,
+/// the row-generation logic is also deterministic.
```

Comment:
What is a deterministic distribution? Is this concept defined somewhere?

Reply:
Added description in the comment.